tests/unit/overlays/test_layers.py

```diff
@@ -95,3 +95,111 @@ def test_save(self, new_dir):
         h = LayerHash(b"some value")
         h.save(part=p1)
         assert Path("parts/p1/state/layer_hash").read_text() == "736f6d652076616c7565"
+
+
+@pytest.mark.usefixtures("new_dir")
+class TestLayerStateManager:
+    """Verify in-memory layer state management operations."""
+
+    def test_get_layer_hash(self):
+        p1 = Part("p1", {})
+        p2 = Part("p2", {})
+        base_layer_hash = LayerHash(b"base hash value")
+
+        Path("parts/p1/state").mkdir(parents=True)
+        lh = LayerHash(bytes.fromhex("a42a1d8ac7fdcfc4752e28aba0b0ee905e7cf96f"))
```

Comment:
bytes are cheap, can we call this var layer_hash and similar for others?

Reply:
Sure, renamed to `layer_hash` (and `p1_layer_hash`, etc).